common/config_parser/section/global_section.py

```diff
@@ -11,8 +14,15 @@ def __init__(self):
         self.logdir = 'Logs'
         self.log_level = 'INFO'
         self.enable_libs_logging = False
+        self._session_log_dir = None
         self.sections = []
 
+    @property
```

Comment:
I just thought do we need it as separate property? Can we calculate it during init right in self.session_log_dir = os.path.join(self.logdir, get_timestamp()) ? Because we don't need to do any validation here